docs/ql-style-guide.md

```diff
@@ -209,6 +211,9 @@ class Type extends ... {
 
   /** ... */
   Type getATypeParameter() { ... }
+  
+  /** ... */
```

Comment:
```suggestion
  /** Gets the HTTP connection ... */
```

Alignment between the acronym in the QLDoc and in the predicate name is one of the reasons I prefer upper-case acronyms. If we go with Pascal case, at least I'd add this example to show that the acronym should still be all upper case in the QLDoc.

Reply:
No matter what we end up going with, I think this points about QLDoc is very good :+1: thanks